src/app/core/metric-module/metric.routes.ts

```diff
@@ -13,8 +13,8 @@ export const METRIC_ROUTES = {
    * Gets learning object metrics
    * @returns metrics for one learning object or all released learning objects
    */
-  GET_LEARNING_OBJECT_METRICS() {
-    return `${environment.apiURL}/learning-objects/metrics`;
+  GET_LEARNING_OBJECT_METRICS(cuid?: string) {
+    return `${environment.apiURL}/learning-objects/metrics?cuid=${cuid}`;
```

Comment:
use querystring to pass `cuid` instead of appending it to the string. this is done so if cuid is empty, we won't pass an empty string which could result in the backend returning nothing

```suggestion
    return `${environment.apiURL}/learning-objects/metrics?${querystring.stringify({cuid: cuid})}`;
```

Reply:
Okay, thanks